bigbluebutton-html5/imports/ui/components/actions-bar/container.jsx

```diff
@@ -1,22 +1,31 @@
-import React, { Component, PropTypes } from 'react';
+import React, { Component } from 'react';
 import { createContainer } from 'meteor/react-meteor-data';
 import ActionsBar from './component';
 import Service from './service';
 import { joinListenOnly } from '/imports/api/phone';
+import { exitAudio } from '/imports/api/phone';
+import { showModal } from '/imports/ui/components/app/service';
+import Audio from '/imports/ui/components/audio-modal/component';
 
 class ActionsBarContainer extends Component {
   constructor(props) {
     super(props);
   }
 
-  render() {
+  openJoinAudio() {
```

Comment:
This dont need to be a class method and you forgot to rename it to `handleJoinAudio`